src/main/java/com/treasuredata/client/TDClientApi.java

```diff
@@ -209,6 +216,22 @@
      */
     String startSavedQuery(String name, Date scheduledTime);
 
+    List<TDSavedQuery> listSavedQueries();
```

Comment:
This interface is missing a `getSavedQuery(String name)` method, but the td rest api doesn't seem to have a method for fetching a single saved query. We should add one.